PackageInfo.g

```diff
@@ -10,8 +10,8 @@ SetPackageInfo( rec(
 
 PackageName := "SmallGrp",
 Subtitle := "The GAP Small Groups Library",
-Version := "1.3",
-Date := "09/04/2018", # dd/mm/yyyy format
+Version := "1.4",
+Date := "09/12/2019", # dd/mm/yyyy format
```

Comment:
Note: this date (presumably September 12) is not in the stated `dd/mm/yyyy` format.

But anyway, I assume that the date will be checked and updated as part of the release process (I'm not sure who does `smallgrp` releases, and how they do them, but presumably automated tools are used that do this sort of thing).

Reply:
Thanks for spotting this -- fixed.

Reply:
yes, there is no need to mix updating dates and code. There may be more commits before the release will be actually wrapped.